cterasdk/core/servers.py

```diff
@@ -10,6 +13,27 @@ class Servers(BaseCommand):
 
     default = ['name']
 
+    def _get_entire_object(self, server):
+        try:
+            return self._portal.get('/servers/%s' % server)
+        except CTERAException as error:
+            raise CTERAException('Failed to retreive server', error)
+
+    def get(self, name, include=None):
+        """
+        Retrieve server properties
+
+        :param str name: Name of the server
+        :param list[str] include: List of fields to retrieve, defaults to ['name']
+        :return: The server, including the requested fields
+        """
+        include = union(include or [], Servers.default)
+        include = ['/' + attr for attr in include]
+        server = self._portal.get_multi('/servers/%s' % name, include)
+        if server.name is None:
+            raise CTERAException('Could not find server', None, name=name)
```

Comment:
In other places the caller can distinguish between the type of errors. For example, to check for 404 (this pattern is used a lot in the Ansible modules).  We should raise something that can be checked programmatically

Reply:
@ygalblum shouldn't we incorporate this across the board then? add a "ObjectNotFound" exception that gets thrown when we can't find an object. Separate task/branch?

Reply:
When the API implementation calls "get" on an object that does not exist, the exception the user receives already includes the 404 code. The problem occurs only when the API implementation calls "get_multi". So, not sure we need to translate all failures to a new exception. Maybe just mimic the 404 exception in cases where get_multi is used and the object was not found

Reply:
Fixed